modules/apps/commerce/commerce-price-list-test/src/testIntegration/java/com/liferay/commerce/price/list/internal/search/test/CommercePriceListIndexerTest.java

```diff
@@ -141,17 +128,28 @@ public void testSearch() throws Exception {
 	@Rule
 	public FrutillaRule frutillaRule = new FrutillaRule();
 
-	private static Company _company;
-	private static Indexer<CommercePriceList> _indexer;
-
-	@Inject
-	private static IndexerRegistry _indexerRegistry;
+	private void _setUserToPermissionChecker(User user) {
+		PermissionThreadLocal.setPermissionChecker(
+			PermissionCheckerFactoryUtil.create(user));
 
-	private static User _user;
+		PrincipalThreadLocal.setName(user.getUserId());
+	}
 
 	@Inject
 	private CommercePriceListLocalService _commercePriceListLocalService;
 
+	@DeleteAfterTestRun
```

Comment:
if the _commercePriceLists are in the company, and the company gets deleted, i dont think we need to try to delete the price lists separately?